recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,19 @@
+:: This works for `setuptools`, but breaks `distutils`.
+:: Will need to figure out a better long term strategy.
+copy "%RECIPE_DIR%\distutils.cfg" "%STDLIB_DIR%\distutils\distutils.cfg"
+if errorlevel 1 exit 1
+
+:: Configure `pip`.
+IF NOT EXIST "%LIBRARY_PREFIX%\etc" mkdir "%LIBRARY_PREFIX%\etc"
+if errorlevel 1 exit 1
+copy "%RECIPE_DIR%\pip.conf" "%LIBRARY_PREFIX%\etc\pip.conf"
+if errorlevel 1 exit 1
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST "%LIBRARY_PREFIX%\etc\conda\%%F.d" MKDIR "%LIBRARY_PREFIX%\etc\conda\%%F.d"
```

Comment:
At least the latest version of the [activation script](https://github.com/conda/conda/blob/master/bin/activate.bat#L51-L56) contains:

```
    @IF EXIST "%CONDA_NEW_ENV%\etc\conda\activate.d" (
```

`%CONDA_NEW_ENV% == %PREFIX%`